docker-compose.yml

```diff
@@ -0,0 +1,53 @@
+version: '2'
+services:
+    gateway:
+        build:
+            context: .
+        volumes:
+            - ./:/go/src/github.com/hellofresh/janus
+        environment:
+            PORT: 3001
+            DEBUG: "true"
+            DATABASE_DSN: "mongodb://mongo:27017"
+            REDIS_DSN: "redis:6379"
+        ports:
+            - "8888:3000"
```

Comment:
Guys this is an open source repo, we can't have this file here.... it shows the auth-service that is private